syncer/db.go

```diff
@@ -267,7 +267,10 @@ func (conn *DBConn) executeSQLWithIgnore(tctx *tcontext.Context, ignoreError fun
 			ret, err := conn.baseConn.ExecuteSQLWithIgnoreError(ctx, stmtHistogram, conn.cfg.Name, ignoreError, queries, args...)
 			if err == nil {
 				cost := time.Since(startTime)
-				txnHistogram.WithLabelValues(conn.cfg.Name).Observe(cost.Seconds())
+				txnHistogram.WithLabelValues(conn.cfg.Name, conn.cfg.WorkerName, conn.cfg.SourceID).Observe(cost.Seconds())
+				// calculate idealJobCount metric: connection count * 1/ (one sql cost time)
+				qps := float64(conn.cfg.WorkerCount) * float64(1) / (cost.Seconds() / float64(len(queries)))
+				idealJobCount.WithLabelValues(conn.cfg.Name, conn.cfg.WorkerName, conn.cfg.SourceID).Set(qps)
```

Comment:
this is QPS so we could merge it with existing QPS panal, with another label, to display them together